grammars/css.cson

```diff
@@ -90,14 +99,14 @@
     ]
   }
   {
-    'begin': '(?=^\\s*@media(\\s+|$))'
+    'begin': '(?=@media(\\s+|$))'
```

Comment:
Hehe, you're totally right.  `(?=@media((?:\\/\\*.\\*\\/)?\\s+|$))` *should* work, but that is suuuuper ugly.